flink-metrics/flink-metrics-influxdb/src/test/java/org/apache/flink/metrics/influxdb/InfluxdbReporterTest.java

```diff
@@ -126,6 +126,8 @@ private MetricRegistryImpl createMetricRegistry(String retentionPolicy) {
 		metricConfig.setProperty(InfluxdbReporterOptions.PORT.key(), String.valueOf(wireMockRule.port()));
 		metricConfig.setProperty(InfluxdbReporterOptions.DB.key(), TEST_INFLUXDB_DB);
 		metricConfig.setProperty(InfluxdbReporterOptions.RETENTION_POLICY.key(), retentionPolicy);
+		metricConfig.setProperty(InfluxdbReporterOptions.CONNECT_TIMEOUT.key(), "10000");
+		metricConfig.setProperty(InfluxdbReporterOptions.WRITE_TIMEOUT.key(), "10000");
```

Comment:
This is not actually needed (and tested anyhow).
Please remove this lines (as they are rather adding noise to the tests), OR add actual tests that verify that http client is properly configured when there are config options.